lib/models/addon-discovery.js

```diff
@@ -11,7 +11,10 @@ const path = require('path');
 const resolve = require('resolve');
 const findup = require('find-up');
 const heimdall = require('heimdalljs');
-
+const PACKAGE_CACHE = Object.create(null);
```

Comment:
I suspect our own tests rely on this. I'll make the cache live as long as the current project, and a cache per project. I think that should cover it?